src/components/CasesPage/CasesPage.scss

```
@@ -0,0 +1,239 @@
+    column-gap: 30px;
+    margin: 0;
+
+    @include breakpoint($desktop-sm) {
+      display: grid;
+      grid-template-columns: 2.4fr 8fr;
+      margin: 0 20px;
+    }
+
+    &-content {
+      width: 100%;
+      max-width: 690px;
+      margin-left: auto;
+      margin-right: auto;
+
+      @include breakpoint($desktop-sm) {
+        margin-left: 0;
+        margin-right: 0;
+      }
+    }
+  }
+
+  &__title {
+    @include font-poppins($fw-bold);
+    @include font-scale(large);
+
+    margin: 0;
+    text-align: center;
+
+    @include breakpoint($phone) {
+      @include font-scale(x3-large);
+    }
+
+    @include breakpoint($tablet-sm) {
+      @include font-scale(x4-large);
+    }
+  }
+
+  &__subtitle {
+    @include font-noto-sans();
+    @include font-scale(x2-medium);
+
+    margin-top: 16px;
+    margin-bottom: 48px;
+    text-align: center;
+  }
+}
+
+.cases-page__cases-list {
+  display: grid;
+  grid-template-columns: repeat(1, 348px);
+  grid-gap: 20px;
+  justify-content: center;
+
+  padding: 72px 0px 48px 0px;
+  margin: 0px 24px 0px 24px;
+  flex-wrap: wrap;
+  align-items: center;
+
+  @include breakpoint($tablet-sm-exact) {
+    grid-template-columns: repeat(2, 348px);
+  }
+
+  @include breakpoint($desktop-sm) {
+    grid-template-columns: repeat(4, 282px);
+    margin: 0px 120px 0px 120px;
+  }
+
+  &-item-box {
+    display: flex;
+    height: 328px;
+    padding: 24px;
+    flex-direction: column;
+    justify-content: flex-end;
+    align-items: flex-start;
+    gap: 24px;
+    border-radius: 16px;
+
+    /* Shadow/Card */
+    box-shadow: 0px 8px 16px 0px rgba(0, 45, 55, 0.12);
+
+    @include breakpoint($desktop-sm) {
+      width: 100%;
+    }
+  }
+
+  &-item {
+    position: relative;
+    display: flex;
+    height: 280px;
+    flex-direction: column;
+    align-items: flex-start;
+    gap: 20px;
+    flex-shrink: 0;
+    align-self: stretch;
+
+    &-trailing {
+      position: absolute;
+      top: 160px;
+      p {
+        color: var(--white);
+        @include font-poppins($fw-semi-bold);
+        font-size: 20px;
+        font-style: normal;
+        font-weight: 600;
+        line-height: 32px; /* 160% */
+      }
+    }
+
+    &-leading {
+      p {
+        margin-top: 0px;
+        color: var(--white);
+        text-align: left;
+        font-feature-settings: 'clig' off, 'liga' off;
+        @include font-poppins($fw-semi-bold);
+        font-size: 14px;
+        font-style: normal;
+        font-weight: 600;
+        line-height: 24px; 
+        letter-spacing: 1px;
+        text-transform: uppercase;
+      }
+    }
+
+    &-icon {
+      width: 96px;
+      height: 96px;
+      flex-shrink: 0;
+      opacity: 0.7;
+    }
+  }
+}
+
+.cases-page__no-more {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-bottom: 120px;
+  > p {
+    color: var(--text-light-grey);
+  }
+  @include breakpoint($tablet-sm-exact) {
+    margin-bottom: 200px;
+  }
+}
```

Please divide with empty lines nested imports and mixins blocks. Please adjust all similar cases.

```suggestion
.cases-page__no-more {
  display: flex;
  justify-content: center;
  align-items: center;
  margin-bottom: 120px;

  > p {
    color: var(--text-light-grey);
  }

  @include breakpoint($tablet-sm-exact) {
    margin-bottom: 200px;
  }
}
```

---

Fixed